### DISCUSSION TITLE
YouTube\Discord

### DISCUSSION THREAD

**Discussion by hetav98** [dovecoteescapee/ByeDPIAndroid#206] [General]
У меня лично этот конфиг работает со всеми операторами, но не работает Telegram, приложения банков и т.д
For me personally, this config works with all operators, but Telegram, banking applications, etc. do not work.

-s1 -q1 -Y -Ar -s5 -o1+s -At -f-1 -r1+s -As -s1 -o1 +s -s-1 -An -b+500

**Comment by AluvianDarkstar** [comment]
У меня внезапно хабр не работает с таким сетапом. Возможно и много других ресурсов не будет. Интересно в чем дело здесь

**Reply by hetav98** [reply]
да, у меня тоже много чего не работает, я просто отключаю когда не нужен
Если кто-то найдёт почему так происходит буду благодарен.

**Reply by Kkekule** [reply]
С этим конфигом и DNS 1.1.1.1 Telegram работает

**Comment by nokiaseries** [comment]
-H:habr.com avito.com -An -npakistan.gov.pk -s1 -q1 -Y -Ar -s2 -o23786+s -f1 -t2 -At -T5 -f-1 -r3+s -t2 -q1 -As

Точно работает на Билайн, Интерсвязь, Ростелеком. После -Н:  можно указать и другие сайты, если они не открываются при VPNe.
